MdeModulePkg/Universal/CapsuleRuntimeDxe/CapsuleService.c

```diff
@@ -4,13 +4,16 @@
   It installs the Capsule Architectural Protocol defined in PI1.0a to signify
   the capsule runtime services are ready.
 
-Copyright (c) 2006 - 2020, Intel Corporation. All rights reserved.<BR>

```

Comment:
We should probably keep the Intel license, right?

Reply:
That was removed in the original code, so I kept it moving forward. I don't see a reason to drop it though.

Reply:
Added back.